features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx

```diff
@@ -49,6 +49,7 @@ const styles = theme => ({
     radioGroup: {
         display: 'flex',
         flexDirection: 'row',
+        marginLeft: theme.spacing.unit * 2,
```

Comment:
This is deprecated in MUI 4.x , Shall we use the below syntax ?

```suggestion
        marginLeft: theme.spacing(2),
```